readme.md

```diff
@@ -287,6 +287,65 @@ See the [AWS section](#aws) for an example.
 
 **Note**: Modifying the `body` is not recommended because the `content-length` header has already been computed and assigned.
 
+###### events
+
+Type: `Object<string, Function[]>`<br>
+Default: `{}`
+
+Events allow listeners to be attached to all events emitted during the request lifecycle. Listeners are subscribed using `on` or `once` according to the known frequency of the event.
+
+The signature for a given listener can be found in the Node.js docs. All listeners will be passed an additional `context` parameter, which is a plain object, that is initialized with an `options` property containing the normalized options for the request.
```

Comment:
> The signature for a given listener can be found in the Node.js docs.

Maybe we can link to at least the main events, so it's quicker for people to look it up.